elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -72,12 +84,12 @@ async def verify_bugs_cli(runtime, verify_bug_status, output, bug_ids):
 
 class BugValidator:
 
-    def __init__(self, runtime: Runtime, output: str = 'text'):
+    def __init__(self, runtime: Runtime, use_jira: bool, output: str = 'text'):
```

Comment:
Let's make the default value False
`use_jira: bool = False` , then we can skip passing false in other places